status-im/status-react
# Use data from API to populate the buy assets sheet

fixes #19836

### Summary
This PR integrates and uses data from the status-go API for the buy assets sheet instead of using dummy data. The only UI change this PR introduces is the change of the title from 'Buy tokens' to 'Buy assets' as per the [design](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=1581-361927&t=DooRzbJqqCsGDIc2-4).

### Testing notes
### Steps to test

- Open Status.
- Switch to the Wallet tab.
- Trigger the buy feature by either:
  - Going into an account and pressing the "Buy" button, or
  - Long pressing on a token and selecting the "Buy" option from the menu.
- Ensure the buy assets sheet is populated.
- Verify that clicking on any asset opens the corresponding page in the external browser.

### Screenshot
<img src="https://github.com/status-im/status-mobile/assets/19279756/b6b640ca-a6a2-42a3-a78d-e9b9abb5eba3" width=350/>


status: ready
